certbot/tests/cli_test.py

```diff
@@ -49,17 +51,30 @@ def setUp(self):
         reload_module(cli)
 
     @staticmethod
-    def parse(*args, **kwargs):
+    def _unmocked_parse(*args, **kwargs):
         """Get result of cli.prepare_and_parse_args."""
         return cli.prepare_and_parse_args(PLUGINS, *args, **kwargs)
 
+    @staticmethod
+    def parse(*args, **kwargs):
+        """Mocks zope.component.getUtility and calls _unmocked_parse."""
+        with test_util.patch_get_utility() as mocked_get_utility: # pylint: disable=unused-variable
```

Comment:
nit: You can get remove the need for the `pylint: disable` if you change this line to:
```
with test_util.patch_get_utility():
```

Reply:
Done.